wcivf/apps/feedback/templates/feedback/feedback_form.html

```diff
@@ -19,24 +22,104 @@ <h5>
             <span aria-hidden="true">🔎</span>
             {% trans "Did you find what you were looking for?" %}
         </h5>
-        {% for choice in feedback_form.found_useful %}
-            {{ choice.tag }}
-            <label class="link-button" for="{{ choice.id_for_label }}">{{ choice.choice_label }}</label>
-        {% endfor %}
-
+        <div id="feedback_choices_container">
+            {% for choice in feedback_form.found_useful %}
+                {{ choice.tag }}
+                <label class="link-button" for="{{ choice.id_for_label }}">{{ choice.choice_label }}</label>
+            {% endfor %}
+        </div>
         <h5>
             <span aria-hidden="true">🗳️</span>
             {% trans "Has this information made you more likely to vote?" %}
         </h5>
-        {% for choice in feedback_form.vote %}
-            {{ choice.tag }}
-            <label class="link-button" for="{{ choice.id_for_label }}">{{ choice.choice_label }}</label>
-        {% endfor %}
+        <div id="voting_choices_container">
+            {% for choice in feedback_form.vote %}
+                {{ choice.tag }}
+                <label class="link-button" for="{{ choice.id_for_label }}">{{ choice.choice_label }}</label>
+            {% endfor %}
+        </div>
 
-        <div>
+        <div id="comments">
             <p>{% trans "Can you tell us anything more?" %}</p>
             {{ feedback_form.comments }}
         </div>
         <button type="submit" class="ds-button-pink">{% trans "Send feedback" %}</button>
     </form>
 </div>
+
+{% block in_page_javascript %}
+    <script>
+        document.addEventListener("DOMContentLoaded", function(event) {
+            var feedback_form = document.forms[1]
```

Comment:
`document.forms[1]` feels fragile here: what if we add another form? This will break silently.

Can you use `getElementById` to get this form?